deps/build.jl

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env julia
+
+path = joinpath(dirname(@__FILE__), "..", "src")
+
+if is_windows()
+    
+    import WinRPM
+    info("Compiling with WinRPM gcc-c++")
+    WinRPM.install("gcc-c++"; yes = true)
+    WinRPM.install("gcc"; yes = true)
+    WinRPM.install("headers"; yes = true)
+    WinRPM.install("winpthreads-devel"; yes = true)
+    
+    gpp = Pkg.dir("WinRPM","deps","usr","x86_64-w64-mingw32","sys-root","mingw","bin","g++")
+    RPMbindir = Pkg.dir("WinRPM","deps","usr","x86_64-w64-mingw32","sys-root","mingw","bin")
+    incdir = Pkg.dir("WinRPM","deps","usr","x86_64-w64-mingw32","sys-root","mingw","include")
+    
+    push!(Base.Libdl.DL_LOAD_PATH, RPMbindir)
+    ENV["PATH"] = ENV["PATH"] * ";" * RPMbindir
+    if success(`$gpp --version`)
+        cd(path) do
+            run(`$gpp -c -fPIC -std=c++11 conv.cpp -I $incdir`)
+            run(`$gpp -shared -o conv.dll conv.o`)
+        end
+    else
+        error("no windows c++ compiler (cl.exe) found, and WinRPM with g++ is failing as well.")
+    end
+    
+end
+
+if is_unix()
+    cd(path) do
+        # Note: on Mac OS X, g++ is aliased to clang++.
+        run(`g++ -c -fPIC -std=c++11 conv.cpp`)
+        run(`g++ -shared -o conv.so conv.o`)
```

Comment:
The default compiler on FreeBSD is clang.
And `cc/c++` is a soft link to it. (But i don't sure `cc/c++` are available on other platform for not

```diff
diff --git a/deps/build.jl b/deps/build.jl
index 995783e..cca00f8 100644
--- a/deps/build.jl
+++ b/deps/build.jl
@@ -31,7 +31,7 @@ end               
 if is_unix()                       
     cd(path) do                    
         # Note: on Mac OS X, g++ is aliased to clang++.                 
-        run(`g++ -c -fPIC -std=c++11 conv.cpp`)
-        run(`g++ -shared -o conv.so conv.o`)
+        run(`c++ -c -fPIC -std=c++11 conv.cpp`)
+        run(`c++ -shared -o conv.so conv.o`)
     end                            
 end
```

Reply:
`c++` works for me on Ubuntu 16.04, will  update.